src/util/logging.cpp

```diff
@@ -1,42 +1,61 @@
 #include <faabric/util/config.h>
 #include <faabric/util/logging.h>
 
-#include <spdlog/sinks/stdout_color_sinks.h>
+#include "spdlog/sinks/basic_file_sink.h"
+#include "spdlog/sinks/stdout_color_sinks.h"
```

Comment:
Why quotes here? I think for "system" libraries/ 3rd party deps we would normally use `<...>`